bitcoin/podle.py

```diff
@@ -9,7 +10,7 @@
 from secp256k1_main import ctx
 PODLE_COMMIT_FILE = None
 N = 115792089237316195423570985008687907852837564279074904382605163141518161494337
-dummy_pub = secp256k1.PublicKey(ctx=ctx)
+dummy_pub = coincurve.PublicKey(None)
```

Comment:
`coincurve.PublicKey(None)` will succeed, but any method calls with fail. This is because anything other than `bytes` is assumed to be a valid `ffi.new('secp256k1_pubkey *')`. This results in a speed improvement for derivation in PrivateKey.